package.json

```diff
@@ -114,7 +113,7 @@
         "src/**/*.ts"
       ],
       "output": [
-        "lib/**"
+        "lib/cjs/*"
```

Comment:
Explicitly point to dir so `wireit` creates the proper cache else every rerun would replace the file leading to a possible bricked state.

Reply:
Don't you want `lib/cjs/**` here, to include subdirectories as well?

Reply:
Ops, that is correct, thank you for noticing! 
```suggestion
        "lib/cjs/**"
```